config.json

```diff
@@ -1169,6 +1183,5 @@
     }
   ],
   "foregone": [
-
```

Comment:
Can you add this blank line back in?

Reply:
Done

Reply:
I looks like you've got some unnecessary white-space here now (see below) - can you please remove it?